src/EpsilonDashboard/main.cpp

```diff
@@ -1,8 +1,30 @@
 #include "EpsilonDashboard.h"
 #include <QApplication>
+#include <QLockFile>
+#include <QDebug>
+#include <QDir>
 
 int main(int argc, char* argv[])
 {
+    QString tmpDir = QDir::tempPath();
+    QLockFile lockFile(tmpDir + "/epsilonDashboard.lock");
+
+    /* Trying to close the Lock File, if the attempt is unsuccessful for 100 milliseconds,
+     * then there is a Lock File already created by another process.
+     * Therefore, we issue a warning and close the program
+     */
+    if (!lockFile.tryLock(100))
+    {
+        qDebug() << "An instance of dashboard already exists." << endl;
```

Comment:
Don't need the trailing `endl` actually, qDebug automatically inserts a newline.